### ISSUE TITLE
Documentation Request: Comparisons with the Keras implementation

### ISSUE THREAD

**Issue by drewm1980** [qubvel/segmentation_models.pytorch#665]
First of all, thanks for gifting such a useful library to the world!

We have been using the Keras implementation (https://github.com/qubvel/segmentation_models) commercially for some time, and would like to migrate to the pytorch implementation with minimal disruption to existing customers. Before doing this we want to build some confidence that the pytorch and keras implementations are equivalent (enough) through testing. Have you already done any equivalence tests (for speed and/or correctness) between your Keras and pytorch implementations, and if so you could provide information regarding what testing was done? 

If there's a test suite for this in a separate repo that we could use as a starting point (and potentially contribute back to), that would be very interesting.

Thanks!
Andrew

**Comment by drewm1980** [comment]
I exported models using both the segmentation_models keras implementations and the pytorch implementations to files I could load into netron.app for visual comparison. I have attached svg and png files for the smallest unet. It is now clear to me that this was not a strict 1:1 port. The conv layer sizes match up, but there are skip connections, batch normalizations, activations, etc... in different places.

![UNET_RESNET18_from_keras h5](https://user-images.githubusercontent.com/190617/197190472-0d661042-231e-4636-8617-3f37f64d169b.png)
![UNET_RESNET18_from_keras h5](https://user-images.githubusercontent.com/190617/197190483-bd5f59ee-69b1-446d-aa32-3eb1691ee77e.svg)
![UNET_RESNET18_from_pytorch pt](https://user-images.githubusercontent.com/190617/197190488-e9fbba62-14a9-4be2-aed9-d492fcc9f594.png)
![UNET_RESNET18_from_pytorch pt](https://user-images.githubusercontent.com/190617/197190492-2287c177-ba02-4024-b233-333ac3e55528.svg)
